converter/shieldhit/beam.py

```diff
@@ -99,6 +99,9 @@ def __str__(self) -> str:
 class BeamConfig:
     """Class mapping of the beam.dat config file."""
 
+    particle: int = 2
+    heavy_ion_a: int = 12
+    heavy_ion_z: int = 6
```

Comment:
```suggestion
    particle: int = 25
    heavy_ion_a: int = 12
    heavy_ion_z: int = 6
```

The combination of `particle_id = 2` (which means proton) doesnt make sense to be used together with Z=6,A=12 (carbon ion)